cypress/support/ServicePageTestHelpers.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeRequestParams, ScheduleDay } from '../../app/models';
+
+export class ServicePageTestHelpers {
+  SERVICE_TITLE = '[data-cy="service-page-title"]'
+  SECTION_ABOUT = '[data-cy="service-about-section"]'
+  SECTION_DETAILS = '[data-cy="service-details-section"]'
+  SECTION_CONTACT = '[data-cy="service-contact-section"]'
+  SECTION_HOURS = '[data-cy="service-loc-hours-section"]'
+  SECTION_HOURS_ROWS = '[data-cy="opening-times-row"]'
+
+  BUTTON_PRINT = '.action-sidebar--print'
```

Comment:
Keeping as is because this component is still jsx and also used by Org page. Will update when they are both updated.